CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ disallow_intree_builds()
 project (utf8proc C)
 
 # Be sure to also update these in Makefile!
-set(SO_MAJOR 3)
-set(SO_MINOR 0)
+set(SO_MAJOR 2)
```

Comment:
ABI compatibility is the question "if we drop the new library in where the old one used to be will things keep working" and because of those changes, I think the answer is no.